fe/fe-core/src/test/java/com/starrocks/storagevolume/StorageVolumeTest.java

```diff
@@ -229,4 +257,186 @@ public void testHDFSInvalidCredential() {
             Assert.assertTrue(e.getMessage().contains("Storage params is not valid"));
         }
     }
+
+    @Test
+    public void testCreate() throws DdlException, AnalysisException {
+        StorageVolumeMgr storageVolumeMgr = GlobalStateMgr.getCurrentState().getStorageVolumeMgr();
+        ExceptionChecker.expectThrowsNoException(
+                () -> {
+                    String sql = "CREATE STORAGE VOLUME IF NOT EXISTS storage_volume_create type = s3 " +
```

Comment:
Can you implement these cases in sql-test?

Reply:
> Can you implement these cases in sql-test?

Reply:
> Can you implement these cases in sql-test?

But if these cases are moved to sql-test, the FE unit tests coverage will fail.

Reply:
> > Can you implement these cases in sql-test?
> 
> But if these cases are moved to sql-test, the FE unit tests coverage check will fail.

make a copy to sql-test and even add more comprehensive cases.

Reply:
We will count coverage after executing sql test. So the coverage is not a solid blocker. UT should cover code that sql test is not easy to cover.

Reply:
> We will count coverage after executing sql test. So the coverage is not a solid blocker. UT should cover code that sql test is not easy to cover.

I have moved the tests to sql test. And there is only 25% FE UT coverage now.